porch/engine/pkg/engine/render.go

```diff
@@ -65,7 +65,7 @@ func (m *renderPackageMutation) Apply(ctx context.Context, resources repository.
 		Type: "eval",
 		Eval: &api.FunctionEvalTaskSpec{
 			Image:     "render",
-			ConfigMap: map[string]string{},
+			ConfigMap: nil,
```

Comment:
This is subtle ... Turns out `map[string]string{}` != `nil`, at least as far as reflect.DeepEqual is concerned.

Reply:
Not surprising, but yes, subtle.